terraform/alerting/alerts.tf

```diff
@@ -253,3 +253,85 @@ resource "google_monitoring_alert_policy" "CloudSchedulerJobFailed" {
     null_resource.manual-step-to-enable-workspace,
   ]
 }
+
+resource "google_monitoring_alert_policy" "HumanAccessedSecret" {
+  count = var.alert_on_human_accessed_secret ? 1 : 0
+
+  project      = var.project
+  display_name = "HumanAccessedSecret"
+  combiner     = "OR"
+
+  conditions {
+    display_name = "A non-service account accessed a secret."
+
+    condition_monitoring_query_language {
+      duration = "60s"
+
+      query = <<-EOT
+      fetch audited_resource
+      | metric 'logging.googleapis.com/user/human_accessed_secret'
+      | align rate(5m)
+      | every 1m
+      | group_by [resource.project_id],
+          [value_human_accessed_secret_aggregate: aggregate(value.human_accessed_secret)]
```

Comment:
Nit: I usually name it `val`, and then you can write `condition val > 0`.

```
| group by [resource.project_id], [val: aggregate(value.human_accessed_secret)]
```

Reply:
Looks nicer your way 😄 . Done